src/sklearn_evaluation/plot/feature_ranking.py

```diff
@@ -197,6 +199,7 @@ def feature_ranks_custom_algorithm(self, ranks):
         return self.ax
 
 
+@apply_theme()
```

Comment:
both this one and the 2D version are no longer displaying the docs correctly: https://sklearn-evaluation--279.org.readthedocs.build/en/279/api/plot.html#rank1d

perhaps it'll be fixed if we apply the decorator to RankD or if we apply it to the method that creates the plot?